packages/grafana-data/src/transformations/transformers/calculateField.ts

```diff
@@ -46,6 +63,12 @@ const defaultReduceOptions: ReduceOptions = {
   reducer: ReducerID.sum,
 };
 
+const defaultWindowOptions: WindowOptions = {
+  reducer: ReducerID.mean,
+  type: WindowType.Trailing,
+  windowSizePercent: 10,
```

Comment:
i would use a `window: { mode: "percent", size: 0.1 }` or something, so we can add more in future